File: Exercises/answer_keys/cpp/pentagon.cpp
#include <iostream>

using namespace std;

int pentagonal(int num){
    int sum = 0;
    int sum2 = 1;

    for(int i = 0; i < num; i++){
        sum2 += sum;
        sum += 5;
    }

    return sum2;
}

int main(){
    int num;

    cout << "Digit a number: ";
    cin >> num;

    cout << "The pentagonal number is: " << pentagonal(num) << endl;

    return 0;
}

File: Exercises/answer_keys/python/pentagon.py
def pentagonal(num):

    sum = 0
    sum2 = 1

    for i in range(num):
        sum2 += sum
        sum += 5

    return sum2

def main():

    num = int(input("Digit a number: "))
    print("The pentagonal number is: ", pentagonal(num))

if __name__ == "__main__":
    main()